src/Provider/Keycloak.php

```diff
@@ -154,6 +168,23 @@ private function getBaseLogoutUrl()
         return $this->getBaseUrlWithRealm() . '/protocol/openid-connect/logout';
     }
 
+    private function fetchIntrospectToken(AccessToken $token) {
```

Comment:
Please fix your code, to same coding style (opening bracet on new line) and fill PHPDoc